src/test/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTripTest.java

```diff
@@ -431,12 +570,22 @@ public String toString() {
      * will be the offset that is applied to all other times in the itinerary.
      */
     private static void updateBaseItineraryTime(Itinerary mockItinerary, ZonedDateTime baseZonedDateTime) {
-        long offset = baseZonedDateTime.toEpochSecond() * 1000 - mockItinerary.startTime.getTime();
-        mockItinerary.startTime = new Date(mockItinerary.startTime.getTime() + offset);
-        mockItinerary.endTime = new Date(mockItinerary.endTime.getTime() + offset);
+        offsetItineraryTime(
+            mockItinerary,
+            baseZonedDateTime.toEpochSecond() * 1000 - mockItinerary.startTime.getTime()
+        );
+    }
+
+    /**
+     * Offsets the itinerary's timing by adding the given offset to the overall start/end time and each leg start/end
+     * times.
```

Comment:
Is this used to inject mock delays? If so, the javadoc should state that.

Reply:
No, delays are stored in a separate field within the legs. This changes the overall time and would preserve any existing delay information.